vars/main-tar.yml

```diff
@@ -0,0 +1,28 @@
+---
+elasticsearch_required_libs:
```

Comment:
I would keep a consistent naming between double underscored vars and normal vars under 'vars' directory :)

Reply:
The double underscored variables are because could be overriden on defaults.yml. This var cannot be override in defaults.yml.
I can make that override on defaults.yml but I'm not sure if is the correct behavour.